crates/cxx-qt-gen/src/generator/cpp/property/mod.rs

```diff
@@ -32,19 +32,27 @@ pub fn generate_cpp_properties(
         let cxx_ty = syn_type_to_cpp_type(&property.ty, type_names)?;
 
         generated.metaobjects.push(meta::generate(&idents, &cxx_ty));
-        generated
-            .methods
-            .push(getter::generate(&idents, &qobject_ident, &cxx_ty));
-        generated
-            .private_methods
-            .push(getter::generate_wrapper(&idents, &cxx_ty));
-        generated
-            .methods
-            .push(setter::generate(&idents, &qobject_ident, &cxx_ty));
-        generated
-            .private_methods
-            .push(setter::generate_wrapper(&idents, &cxx_ty));
-        signals.push(signal::generate(&idents, qobject_idents));
+
+        if let Some(getter) = getter::generate(&idents, &qobject_ident, &cxx_ty) {
+            generated.methods.push(getter);
+            generated
+                .private_methods
+                .push(getter::generate_wrapper(&idents, &cxx_ty));
```

Comment:
if the `generate_wrapper` also returns Option then this should be put into another if block like the setter and then have an else block with the unreachable too etc

Reply:
also i guess ignore this as `getter_wrapper` is `Name`

Reply:
changed